Mark strings for translation in templates/zerver

shumbashi opened the pull request:
In order to enable internationalization support in Zulip, and to use Django internationalization tools, all strings in Zulip frontend needs to be marked for translation.

smarx commented:
*Automated message from Dropbox CLA bot*

@shumbashi, it looks like you've already signed the Dropbox CLA. Thanks!

shumbashi commented:
Please note that for the tests to pass it depends on #253

shumbashi commented:
Closed and re-opened to re-run travis tests after #253 was merged into master.

timabbott commented:
Cool, thanks for doing this @shumbashi !!  A few comments:

* It looks like there's some style inconsistency with not having a space before the closing %}, which I think we should fix.  You can I think find all the instances using  `git grep '"%}'` and I think it should be an easy sed job to fix them all.  
* The commit message second paragraph should be line-wrapped.
* We can probably just skip translating the API docs for now since a lot of the content is awkwardly in the templates/zerver/api_content.json file, so feel free to just remove the hunks from those two files.

Otherwise this looks pretty good!

shumbashi commented:
@timabbott I have amended this commit to include your comments:
- [X] Fixed inconsistency with '"%}
- [X] Fixed commit message line-wrapping
- [X] Removed i18n note from API templates.

timabbott commented:
Awesome, thanks for the contribution!  Merged.
